fiscal_epos_print_fiscalcode/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,7 @@
+**Italiano**
+
+Nel form del partner, abilitare `Codice fiscale negli scontrini`.
```

Comment:
Grazie per aver aggiunto la sezione in lingua italiana. :-)

s/nel form/nella scheda

In genere in italiano vengono usati i doppi apici " ", in inglese quelli singoli. (vedi https://www.odoo-italia.org/documentazione/12.0/traduzioni/linee_guida/regole_grafiche.html#virgolette)

Reply:
ok grazie ora ne so ancora una in più :)
Prima di applicarlo però ti chiedo un chiarimento: qui ho messo gli apici inversi solo per fare in modo che il nome del campo venisse visualizzato diversamente (in questo caso in corsivo) nel README lato utente.
Quindi non vengono visualizzati gli apici inversi ma il nome del campo è semplicemente in _corsivo_.

Può andare bene o i nomi dei campi vanno proprio racchiusi dai doppi apici?

Reply:
Ops... 😅  
Va benissimo, nella fretta non avevo notato fossero apici inversi. 👍

Reply:
> Grazie per aver aggiunto la sezione in lingua italiana. :-)
> 
> s/nel form/nella scheda

Dopo questo è ok.  👍

Reply:
Fatto